src/BlazorUI/Bit.BlazorUI/Components/Buttons/ToggleButton/BitToggleButton.scss

```
@@ -15,25 +15,128 @@
     border-style: $shp-border-style;
     border-radius: $shp-border-radius;
     font-size: var(--bit-tgb-fontsize);
+    // a gap rather than a margin on the icon, so reversing the icon position keeps the spacing on the right side
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)